### PR TITLE
[Do Not Review]dualstack overlay pipeline

### PR DESCRIPTION
**Pull request by paulyufan2**
This PR will build up dualstack overlay pipeline in ACN Repo. Actively working on this PR.

### CODE REVIEW

**Review by pjohnst5 — COMMENTED** [review]
Just a couple questions

File: .pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  - script: |
+      ls -lah
+      pwd
+      echo "installing kubectl"
+      curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+      sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+      kubectl cluster-info
+      kubectl get node
+      kubectl get po -owide -A
+    name: "installKubectl"
```
[the remark this answers is not in the document]
Comment (paulyufan2) [reply .pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml:51]: Yes we do need install kubectl to get cluster-info, node and so on

**Comment by paulyufan2** [comment]
use one PR to track: https://github.com/Azure/azure-container-networking/pull/2032/files

**Comment by paulyufan2** [comment]
use one PR to track: https://github.com/Azure/azure-container-networking/pull/2032/files

**Pull request closed without merging**